araig_utils/package.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<package>
+    <name>araig_utils</name>
+    <version>0.0.0</version>
+    <description>Utils for ARAIG</description>
+    <maintainer email="changxuan.li@ipa.fraunhofer.de">Changxuan Li</maintainer>
```

Comment:
With this line you are committing to maintain this script even after you finish your HiWi job. Are you willing to do this? ;) If not, put me here. Then it becomes my responsibility to maintain this package.